src/main/java/com/twilio/http/bearertoken/BearerTokenTwilioRestClient.java

```diff
@@ -119,9 +128,15 @@ public Response request(BearerTokenRequest request) {
         }
         logRequest(request);
```

Comment:
During request logging we are printing key of **authorization** which will be equal to "Bearer actual_token_value"
So in short token is getting logged, which shouldn't be logged.
Can you make sure that clientSecret and token is not logged anywhere.

Reply:
You might need to override logRequest method in accordance to bearer token request.